test/uk/gov/hmrc/cipphonenumberverification/services/VerifyServiceSpec.scala

```diff
@@ -16,76 +16,50 @@
 
 package uk.gov.hmrc.cipphonenumberverification.services
 
-import org.mockito.ArgumentMatchers.any
-import org.mockito.Mockito.{verify, when}
-import org.mockito.{Answers, ArgumentCaptor}
+import org.mockito.ArgumentMatchersSugar.any
+import org.mockito.IdiomaticMockito
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
-import org.scalatestplus.mockito.MockitoSugar
 import play.api.http.Status._
-import play.api.libs.json.{JsObject, JsValue, Json}
-import play.api.mvc.Results.{BadRequest, Ok}
 import play.api.test.Helpers.{contentAsJson, defaultAwaitTimeout, status}
 import uk.gov.hmrc.cipphonenumberverification.connectors.{GovUkConnector, ValidateConnector}
 import uk.gov.hmrc.cipphonenumberverification.models.{Passcode, PhoneNumber}
-import uk.gov.hmrc.cipphonenumberverification.repositories.PasscodeCacheRepository
 import uk.gov.hmrc.http.{HeaderCarrier, HttpResponse}
-import uk.gov.hmrc.mongo.cache.DataKey
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
 class VerifyServiceSpec extends AnyWordSpec
   with Matchers
-  with MockitoSugar {
-
-  val validationSuccessfulResponseBody: JsObject = Json.obj("phoneNumber" -> "+447812345678", "phoneNumberType" -> "Mobile" )
-  val validationFailureResponseBody: JsObject = Json.obj("code" -> "VALIDATION_ERROR", "message" -> "Enter a valid telephone number")
-  val govUkSuccessResponseBody: JsObject = Json.obj("id" -> "test-notification-id")
-  private val phoneNumberCaptor = ArgumentCaptor.forClass(classOf[PhoneNumber])
-  private val passcodeArgCaptor = ArgumentCaptor.forClass(classOf[Passcode])
+  with IdiomaticMockito {
 
   "verify" should {
-    "return success if telephone number is valid" ignore new SetUp {
-      when(validateConnectorMock.callService(PhoneNumber("test"))(hc))
-        .thenReturn(Future.successful(HttpResponse(200, validationSuccessfulResponseBody, Map("content-type" -> Seq("application/json")))))
-      val phoneNumber: PhoneNumber = PhoneNumber("+441292123456")
-      val passcode: Passcode = Passcode(phoneNumber = phoneNumber.phoneNumber, otp = "testPasscode")
-      val savedPasscodeFromDb: Passcode = Passcode(phoneNumber = phoneNumber.phoneNumber, otp = passcode.otp)
-      val futurePasscodeFromDb: Future[Passcode] = Future.successful(savedPasscodeFromDb)
-
-      when(passcodeService.persistPasscode(phoneNumberCaptor.capture())(any[HeaderCarrier])).thenReturn(futurePasscodeFromDb)
-
-      val futureGovUkSuccess = Future.successful(Right(HttpResponse(CREATED, govUkSuccessResponseBody.toString())))
-      when(govUkConnectorMock.sendPasscode(any[Passcode])(any[HeaderCarrier])).thenReturn(futureGovUkSuccess)
-
+    "return success if telephone number is valid" in new SetUp {
+      validateConnectorMock.callService(PhoneNumber("test"))
+        .returns(Future.successful(HttpResponse(OK, """{"phoneNumberType": "Mobile"}""")))
```

Comment:
This is incorrect for 2 reasons, 

1. the normalised phoneNumber property is missing and
2.  its used in multiple places so put at top of class or in SetUp

Reply:
since we are mocking the validate connector, it will return whatever value we setup in the mock. And, we are not doing anything with it in the sut

Reply:
yip ok, just make sure the integration tests check for both fields in the response body

Reply:
although i did have it in my code that was merged the other day ago.  i remember adding the phoneNumber: xxx to the tests.  I think we should have both in the tests even although it is part of a mock call.  as it represents what is actually returned by the call?

Reply:
i will do in audit ticket